app/constants.ts

```diff
@@ -25,6 +25,7 @@ export const svg: any = {
   background: document.getElementById('artistic-background'),
   points: document.getElementById('points'),
   triangles: document.getElementById('triangles'),
+  circumCircles: document.getElementById('circum-circles'),
```

Comment:
Been debating with myself whether circumcircle is one word or two. Lets roll with this for now.